### ISSUE TITLE
Destroy, reload

### ISSUE THREAD

**Issue by ricnunes** [codrops/GridLoadingEffects#8]
Hi codrops,

I'm implementing this on a new website and I'm trying to add a filter system to the masonry, but I'm not able to run any of the masonry options, since it's being recreated on AnimeOnScroll. 

Any idea on how I can reload the grid, after hiding the filtered cards?

What I was trying to do, but is currently failing: 

`$li.removeClass('active');
$('#grid').find('.item-'+targetLink).addClass('hidden');
$('#grid').masonry('layout');`

Thank you :)

**Comment by BDominik** [comment]
Did you solved this problem?

**Comment by ricnunes** [comment]
Yes I did. Can't quite remember exactly everything now, but I believe I destroyed and re-created the masonry every time I clicked on a filter option.
